include/world_builder/features/fault_models/grains/random_uniform_distribution_deflected.h

```diff
@@ -0,0 +1,114 @@
+/*
+  Copyright (C) 2018 - 2021 by the authors of the World Builder code.
+
+  This file is part of the World Builder.
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef WORLD_BUILDER_FEATURES_FAULT_MODELS_GRAINS_RANDOM_UNIFORM_DISTRIBUTION_DEFLECTED_H
+#define WORLD_BUILDER_FEATURES_FAULT_MODELS_GRAINS_RANDOM_UNIFORM_DISTRIBUTION_DEFLECTED_H
+
+
+#include "world_builder/features/fault_models/grains/interface.h"
+
+namespace WorldBuilder
+{
+
+  namespace Features
+  {
+    using namespace FeatureUtilities;
+    namespace FaultModels
+    {
+      namespace Grains
+      {
+        /**
+         * This class represents a fault and can implement
```

Comment:
Is this comment block copied from somewhere?

Reply:
Yes, I this is copied from random_uniform_distribution.h in the same folder as a template and I made changes based on that.